src/app/services/services.py

```diff
@@ -0,0 +1,5 @@
+from app.services.bucket.s3 import S3Service
+from app.services.bucket.baseBucketService import BaseBucketService
+
+bucket_service = S3Service()
+assert isinstance(bucket_service, BaseBucketService), "selected service is not an instance of BaseBucketService"
```

Comment:
Is this for debug?

Reply:
It is to make sure the selected bucket service inherits from ̀BaseBucketService`. 
As python is not typed; I don't know how to do it in another way. But if you have a suggestion, I would be glad to hear it, I might have overlooked a very simple solution.

Reply:
But I'm not sure we need to :sweat_smile: 
Since you create your `bucket_service` with a class that inherits from BaseBucketService, I'm not seeing why we need to ?
If someone does create the bucket_service another way, then they won't be able to call its methods properly. But that makes sense. That check would be good to put in our tests, but I don't see it as something to put in this module

Reply:
Haha indeed. I moved it to a unitTest